task_01.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" Docstring for fahrenheit temp to celsius"""
+
+
+import decimal
+
+decimal.getcontext().prec = 5
+
+ABSOLUTE_DIFFERENCE = decimal.Decimal(273.15)
```

Comment:
You were _so close_! Remember that `273.15` is a _float_ -- even when you wrap it in a decimal `decimal.Decimal(273.15)` -- to achieve the precision you needed you had to establish the decimal with a string value, eg: `decimal.Decimal('273.15')`